### ISSUE TITLE
Use esm

### ISSUE THREAD

**Issue by skellock** [infinitered/gluegun#320]
Regarding some of your valid concerns in #319.

ES modules is coming to Node!  👍 
But not yet.  👎
And only to `*.mjs`. 👎 
But what the js community wants, the [js community gets](https://github.com/standard-things/esm).  👍 

TURBO JAMON MODE ENGAGE!!!!

**Comment by jamonholmgren** [comment]
I've done a bunch of research on this. Given how much in flux all of this is right now, I think I'd stick to commonjs `require`s where we need to and just deal with it. Once it all shakes out (probably in 2018, I'd imagine) then we can adopt whatever standard comes along. Thoughts?

**Comment by skellock** [comment]
Up to you.

**Comment by jamonholmgren** [comment]
@arackaf [thinks it should be easy](https://twitter.com/AdamRackis/status/969787589606760449) so I'm re-opening this.

**Comment by arackaf** [comment]
Yeah! As the code linked shows, I'd highly recommend using @std/esm. cjs and native Node interop is seamless, and the outside world can consume your Node module with regular old require()

**Comment by mikenikles** [comment]
So... ESM has come a long way 😄. We are considering to fork Gluegun for https://github.com/webstoneHQ/webstone and migrate to ESM so we can better leverage ESM-only NPM packages.

Is there interest in the Gluegun community to instead migrate the official Gluegun package to ESM? @cahllagerfeld and I could migrate our fork and contribute it back to this repo here.
